### PR TITLE
Zds 357/search update zds lib to new designs

### PR DESCRIPTION
**Pull request by cabutler10**

### CODE REVIEW

File: src/components/__tests__/__snapshots__/Header.spec.js.snap [hunk 1]
```diff
@@ -123,30 +123,89 @@ c0,0.1,0.1,0.2,0.1,0.4L115.4,16.3L115.4,16.3z"
               </svg>
             </div>
             <div
-              class="zep-grid__col zep-grid__col--align-self-center zep-grid__col--xxs-1-4 zep-grid__col--xs-1-6 zep-grid__col--m-2-8"
+              class="zep-grid__col zep-grid__col--align-self-center zep-grid__col--xxs-0-4  zep-grid__col--m-6-8 Header-searchLarge-0-1-5"
```
Comment (byjs-dev) [inline src/components/__tests__/__snapshots__/Header.spec.js.snap]: .zep-grid__col--xxs-0-4 is not possible only 1 - 4 of 4 and you have to include col--xs as well between xxs and m

File: src/components/__tests__/__snapshots__/Header.spec.js.snap [hunk 2]
```diff
@@ -289,30 +403,89 @@ c0,0.1,0.1,0.2,0.1,0.4L115.4,16.3L115.4,16.3z"
             </svg>
           </div>
           <div
-            class="zep-grid__col zep-grid__col--align-self-center zep-grid__col--xxs-1-4 zep-grid__col--xs-1-6 zep-grid__col--m-2-8"
+            class="zep-grid__col zep-grid__col--align-self-center zep-grid__col--xxs-0-4  zep-grid__col--m-6-8 Header-searchLarge-0-1-5"
```
Comment (byjs-dev) [inline src/components/__tests__/__snapshots__/Header.spec.js.snap]: ...same here

File: src/content/code/searchReact.mdx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        isOpen,
+        inputValue,
+        highlightedIndex,
+        selectedItem,
+      }) => (
+        <div className={classnames(classes.container, 'zep-form-container')}>
+          <label {...getLabelProps()} className="zep-visually-hidden">
+            {placeholder}
+          </label>
+          <input {...getInputProps()} className="zep-search" />
```
Comment (byjs-dev) [inline src/content/code/searchReact.mdx]: ...same here in the example: .zep-search should be on a parent container...

**Comment by cabutler10** [comment]
@byjs-dev Please review ZEL bugfix first and I will publish that to npm then push another commit to this branch to update zeppelin-element-library before we merge this, thanks!

**Comment by byjs-dev** [comment]
Ok, review ZEL bugfix done! ;) You can go on.

**Pull request merged**